liquid_node/commands.py

```diff
@@ -281,6 +282,15 @@ def start(job, hcl):
         tokens = json.loads(run(docker_exec_cmd, shell=False))
         vault.set(app['vault_path'], tokens)
 
+    # check if there are jobs to stop
```

Comment:
The `deploy` command shuoldn't stop anything, we have separate commands for stopping things (gc).

Reply:
I disagree, the deploy command should bring the system to the state that corresponds to the configuration from the ini file. If after deploy, the system state does not correspond to the configuration from the ini file then this is confusing.

Reply:
In principle you are right, but deploy should never ever stop anything that it did not start itself. That's a tricky problem to solve and is not worth our time right now.

Reply:
All jobs are started by deploy, there is no other command that starts jobs

Reply:
Also, there's nothing tricky about stopping some jobs for the apps that were disabled